### ISSUE TITLE
Shortlinks for cheatsheets

### ISSUE THREAD

**Issue by tghosth** [OWASP/CheatSheetSeries#1300]
Does the cheatsheet site have a mechanism for shorter links, e.g.

https://cheatsheetseries.owasp.org/SSRF

would redirect to:

https://cheatsheetseries.owasp.org/cheatsheets/Server_Side_Request_Forgery_Prevention_Cheat_Sheet.html

**Comment by jmanico** [comment]
We do not have that set up but it would be easy to do so.

**Comment by szh** [comment]
Yes, we could add this in https://github.com/OWASP/CheatSheetSeries/blob/master/scripts/Generate_Site_mkDocs.sh#L98, though we should probably refactor this into something more maintainable.

**Comment by tghosth** [comment]
Seems like it needs a loop or something there...

**Comment by sambhavnoobcoder** [comment]
hey @tghosth @szh  , is this issue still open ? I would like to contribute to it .

**Comment by szh** [comment]
@sambhavnoobcoder unless @tghosth confirms he's working on it, you're more than welcome to take this.

**Comment by tghosth** [comment]
No I would be delighted for @sambhavnoobcoder to work on this :)

**Comment by sambhavnoobcoder** [comment]
okay thanks !! I'll start working on this right away .
